roles/api-environment/tasks/express-api.yml

```diff
@@ -1,4 +1,20 @@
 ---
+  - name: "create docker script directory"
+    file:
+      path: "{{ express_api_docker_path }}"
+      state: directory
+      owner: root
+      group: root
```

Comment:
Is this task needed? Since you're using `copy` module later, base on the Ansible 2.8 document [here](https://docs.ansible.com/ansible/latest/modules/copy_module.html):

> If dest is a non-existent path and if either dest ends with "/" or src is a directory, dest is created.

Can you confirm it's correct?